crates/runtime/src/jit_int.rs

```diff
@@ -16,6 +20,10 @@ const JIT_NOACTION: u32 = 0;
 const JIT_REGISTER_FN: u32 = 1;
 const JIT_UNREGISTER_FN: u32 = 2;
 
+lazy_static::lazy_static! {
```

Comment:
I've worked pretty hard to remove `lazy_static` from most crates because it often behaves in susprising ways. I don't understand much about the GDB integration here, but I think it would be better to have an instanced lock and/or atomics to manage this.

Reply:
gdb will use `__jit_debug_descriptor` (see static below) on this module startup and during reactiving on `__jit_debug_register_code` breakpoint AFAIK. The `__jit_debug_descriptor` is a global symbol for entire module.

Reply:
Is it safe to assume there won't be any concurrent access here?